src/storage/v2/transaction.hpp

```diff
@@ -26,24 +27,28 @@
 #include "storage/v2/vertex.hpp"
 #include "storage/v2/vertex_info_cache.hpp"
 #include "storage/v2/view.hpp"
+#include "utils/bond.hpp"
+#include "utils/pmr/list.hpp"
 
 namespace memgraph::storage {
 
 const uint64_t kTimestampInitialId = 0;
 const uint64_t kTransactionInitialId = 1ULL << 63U;
+using PmrListDelta = utils::pmr::list<Delta>;
```

Comment:
Thinking about space saving (maybe in a later PR), would forward_list be worth using? Do we use reverse iteration?

Also thinking of further work on this theme, it would be nice to have a special kind of skip list for destruction. We only need to destruct the deltas which are not trivially destructible. ie. track Action::DELETE_DESERIALIZED_OBJECT and Action::SET_PROPERTY deltas for this purpose.

Reply:
Yes, it makes sense. I will open other PR once I find time to investigate this.